src/sentry/discover/endpoints/discover_key_transactions.py

```diff
@@ -56,9 +68,12 @@ def get(self, request, organization):
 
         queryset = KeyTransaction.objects.filter(organization=organization, owner=request.user)
 
+        if queryset.count() == 0:
```

Comment:
```suggestion
        if not queryset.exists():
```

is this better?

Reply:
Yeah, good catch. Exists is more efficient than count